reana-ui/src/actions.js

```diff
@@ -2,7 +2,7 @@
 	-*- coding: utf-8 -*-
 
 	This file is part of REANA.
-	Copyright (C) 2019 CERN.
+	Copyright (C) 2020 CERN.
```

Comment:
Actually it should be `2019, 2020` since we keep previous years in the copyright messages.

(No need for action now, we can amend that programmatically one day...)